app/__init__.py

```diff
@@ -202,7 +202,7 @@ def add_blueprints(application):
 def setup_secure_cookies(application):
     application.secret_key = settings.EQ_SECRET_KEY
     application.permanent_session_lifetime = timedelta(
-        seconds=settings.EQ_SESSION_TIMEOUT_SECONDS)
+        seconds=settings.EQ_SESSION_TIMEOUT_SECONDS + settings.EQ_SESSION_TIMEOUT_GRACE_PERIOD_SECONDS)
```

Comment:
... Making sure that the client always times-out with a buffer (30 seconds?) before the server to account for delays etc.